afl-fuzz.c

```diff
@@ -1349,6 +1350,13 @@ static void cull_queue(void) {
         weight *= slow_fac;
       }
     }
+    if (enable_boost_fast_seqs) {
+      double base_weight_fac = 2.0;
+      double max_weight_fac_decr = 1.75;
+      double scale_fac = 0.01;
+      double execs_per_sec = 0.000001 / (double) q->exec_us;
```

Comment:
Actually, I think I got this wrong... Maybe it should be `1000000.0 / (double) q->exec_us`. Do you agree?

Reply:
Oh yeah!, it should be `1000000.0 / (double) q->exec_us` to find executions per second from the given microseconds.